behave/Dockerfile

```diff
@@ -25,4 +25,7 @@ COPY . /behave
 
 COPY ./wrapper.sh wrapper.sh
 
-ENTRYPOINT ["./wrapper.sh"]
+ARG a11y="false"
+ENV run_a11y $a11y
+
+ENTRYPOINT ./wrapper.sh $run_a11y
```

Comment:
```ARG a11y="false"
ENV run_a11y $a11y

ENTRYPOINT ./wrapper.sh $run_a11y```
I thought we removed adding the a11y stuff to build and it's now controlled through an environment variable?

Reply:
Removed, thank you